paparazzi/src/test/java/app/cash/paparazzi/helper/TestSnapshotVerifier.kt

```diff
@@ -0,0 +1,36 @@
+package app.cash.paparazzi.helper
+
+import app.cash.paparazzi.Snapshot
+import app.cash.paparazzi.SnapshotHandler
+import app.cash.paparazzi.internal.ImageUtils
+import org.junit.rules.TemporaryFolder
+import java.awt.image.BufferedImage
+import java.io.File
+import javax.imageio.ImageIO
+
+internal class TestSnapshotVerifier(val tempDir: TemporaryFolder) : SnapshotHandler {
+  override fun newFrameHandler(
+    snapshot: Snapshot,
+    frameCount: Int,
+    fps: Int
+  ): SnapshotHandler.FrameHandler {
+    return object : SnapshotHandler.FrameHandler {
+      override fun handle(image: BufferedImage) {
+        val expected = File("src/test/resources/${snapshot.name}.png")
+        val failed = File("src/test/resources/${snapshot.name}/failed.png")
+        ImageUtils.assertImageSimilar(
+          relativePath = expected.path,
+          image = image,
+          goldenImage = ImageIO.read(expected),
+          maxPercentDifferent = 0.1,
+          relativePathFailure = failed.path,
+          failureDir = tempDir.newFolder()
```

Comment:
```suggestion
          failureDir = failureDir
```

per the comment in this same source file; however, I'd really prefer not to pass this into assertImageSimilar